Parsers/ASimAuditEvent/Tests/Microsoft_GithubWebhook_ASimAuditEvent_DataTest.csv

```diff
@@ -0,0 +1,11 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 46 records (100.0%) for field [EventProduct] of type [Enumerated]: [""GithubWebhook""] (Schema:AuditEvent)"
+"(1) Warning: Empty value in 1 records (2.17%)  in mandatory field [EventResult] (Schema:AuditEvent)"
+"(1) Warning: Empty value in 1 records (2.17%)  in mandatory field [EventType] (Schema:AuditEvent)"
+"(1) Warning: Empty value in 24 records (52.17%)  in mandatory field [Operation] (Schema:AuditEvent)"
```

Comment:
Mandatory fields cannot be empty. Please see in which cases these are coming as empty and derive Operation value for those cases.